ignite/cmd/plugin.go

```diff
@@ -360,6 +364,130 @@ func NewPluginUpdate() *cobra.Command {
 	}
 }
 
+func NewPluginAdd() *cobra.Command {
+	cmdPluginAdd := &cobra.Command{
+		Use:  "add [path]",
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return fmt.Errorf("must provide a plugin path")
+			}
+			chain, err := newChainWithHomeFlags(cmd)
+			s := cliui.New(cliui.WithStdout(os.Stdout))
+
+			if err != nil {
+				return err
+			}
+			nodePath := chain.ConfigPath()
+			nodePath = path.Dir(nodePath)
+
+			confPath, err := pluginsconfig.LocateDefault(nodePath)
+			if err != nil {
+				return err
+			}
+			var conf pluginsconfig.Config
+			f, err := os.Open(confPath)
+			if err != nil {
+				return err
+			}
+			err = conf.Decode(f)
+			if err != nil {
+				return err
+			}
+
+			for _, p := range conf.Plugins {
+				if p.Path == args[0] {
+					return fmt.Errorf("cannot add duplicate plugin %s", args[0])
+				}
+			}
+
+			p := pluginsconfig.Plugin{
+				Path: args[0],
+			}
+
+			ctx := context.Background()
+			s.StartSpinner("Loading plugins")
+			pluginInstance, err := plugin.LoadSingle(ctx, &p)
+			s.StopSpinner()
+
+			if err != nil {
+				return err
+			}
+			if pluginInstance.Error != nil {
+				return errors.Wrapf(pluginInstance.Error, fmt.Sprintf("Error while attempting to load plugin: %s\n", args[0]))
+			}
+			s.Println("Done loading plugin")
+
+			conf.Plugins = append(conf.Plugins, p)
+			s.Printf("🎉 %s added \n", args[0])
+			err = conf.Save(confPath)
+
+			if err != nil {
+				return err
+			}
```

Comment:
```suggestion
			err = conf.Save(confPath)
			if err != nil {
				return err
			}
			s.Printf("🎉 %s added \n", args[0])
```

Let's celebrate after the config is saved :)